### DISCUSSION TITLE
Tuya Web Platform Plugin Help

### DISCUSSION THREAD

**Discussion by realdeal1974** [milo526/homebridge-tuya-web#576] [Q&A]
Hi All,  I installed the above plugin but my Heat Storm Wall Heater shows in HomeKit with no response at all times.  My other Tuya smart plugs all work.  Any suggestions?  Thanks!

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

**Comment by milo526** [comment]
Closing since there was no response
